### DISCUSSION TITLE
authorization in GROUP share level

### DISCUSSION THREAD

**Discussion by user-GitHub-user** [apache/kyuubi#4781] [Q&A]
hello, i use ranger for authorization, in group share level i want fine-grained access control for session user, I don't quite understand the meaning in the document. Do I need to modify the code of kyuubi to support this function, or should I modify the code of spark? Does kyuubi have a configuration to support it directly?
![image](https://user-images.githubusercontent.com/18843801/235089319-229c0905-cf46-4f92-ade5-2baf469844d9.png)

**Comment by yaooqinn** [comment]
AFAICR, kyuubi session user will be populated by the Kyuubi server automatically

**Comment by HaoYang670** [comment]
In my cases (I use Ranger), the policy defined for kyuubi session users could work. But those defined for groups doesn't, even if I had use the primary group name to create a user in Ranger and defined policies for them
